catalogs/catalog-common/build.gradle.kts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2023 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+
+plugins {
+  id("java")
+}
+
+group = "org.example"
+version = "0.5.0-SNAPSHOT"
```

Comment:
Why do you need to hardcode this version, and what is "org.example"?

Reply:
Sorry, these two are auto-generated by IDE, I will handle them.